.golangci.yml

```diff
@@ -46,3 +46,6 @@ linters:
 #       - wrapperFunc
 #       - commentFormatting # https://github.com/go-critic/go-critic/issues/755
 
+service:
+  golangci-lint-version: 1.17.x # use the fixed version to not introduce new linters unexpectedly
+  prepare:
```

Comment:
doing this now, will go through and update to most recent and clean up according to the added linters in the coming days.